### ISSUE TITLE
Tensorflow Serving of Inception V3 Scores need explaination

### ISSUE THREAD

**Issue by jyotiskab** [tensorflow/serving#160]
I have been trying to get an idea of how exactly I should interpret the scores as returned from Tensorflow Serving Inference model for Inception v3. I have read all documentations and explored the code base, but as of now, I don't understand the scores generated. Perhaps someone can guide me in the right direction. As mentioned in the tutorial for TF Serving with Kubernates w/ docker: https://tensorflow.github.io/serving/serving_inception.html , the following scores are generated. 

> $ bazel-bin/tensorflow_serving/example/inception_client --server=146.148.88.232:9000 --image=/path/to/my_cat_image.jpg
> scores: 9.41664886475
> scores: 8.14928436279
> scores: 7.6565990448
> scores: 3.85941624641
> scores: 2.82326698303
> classes: "tiger cat"
> classes: "cougar, puma, catamount, mountain lion, painter, panther, Felis concolor"
> classes: "Persian cat"
> classes: "leopard, Panthera pardus"
> classes: "Egyptian cat"

The scoring methodology here seems to be different than it is in here: https://www.tensorflow.org/versions/r0.10/tutorials/image_recognition/index.html

As we see in the recognition tutorial, the scores make much more sense. 

Would someone please elaborate on the scoring methodology used in the Serving model, and how can I change the scoring to that of image recognition?

**Comment by LiberiFatali** [comment]
Using softmax function on returned scores will convert them to probability as in that tutorial.

**Comment by jyotiskab** [comment]
Ah, gotcha. I realized that the classify_image.py was using a softmax tensor, wasn't sure if softmax was applicable for the TF Serving model.  Thank you.

**Comment by montanaflynn** [comment]
@jyotiskab could you share what you modified to get the probabilities instead of the scores in TF Serving?
